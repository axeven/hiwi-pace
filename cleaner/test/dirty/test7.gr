p tw 11 11
1 2
1 4
2 3
3 4
4 5
5 9
6 7
6 8
7 8
9 10
9 11
