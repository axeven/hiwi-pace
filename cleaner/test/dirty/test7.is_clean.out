test7.gr dirty 11 11 not connected 0 2 7 2
